project/app/forms.py

```diff
@@ -4,4 +4,4 @@
 class EditGroupForm(forms.ModelForm):
     class Meta:
         model = Group
-        fields = ('name', 'maxsize', 'course_code', 'type', 'students',)
\ No newline at end of file
+        fields = ('students',)
```

Comment:
En oo varma oonko muistanu tästä sanoa, mut jokaisen tiedoston lopussa pitäis olla yks tyhjä rivinvaihto. Sitä tässäkin meinaa toi punainen rivinvaihtonuoli.